contrib/src/main/java/com/datatorrent/contrib/cassandra/AbstractCassandraInputOperator.java

```diff
@@ -45,7 +47,8 @@
 public abstract class AbstractCassandraInputOperator<T> extends AbstractStoreInputOperator<T, CassandraStore> {
 
   private static final Logger logger = LoggerFactory.getLogger(AbstractCassandraInputOperator.class);
-
+  private PagingState nextPageState;
+  private int fetchSize;
   int waitForDataTimeout = 100;
   @AutoMetric
   protected long tuplesRead;
```

Comment:
The reason we decided to add it was, if a operator dies, the count provided as tuples count may not be right. Do we have any better solution to address this?